packages/valora-auth/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@wallet/valora-auth",
+  "version": "1.0.0",
+  "description": "Access Valora API keys",
+  "author": "Celo",
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "serve": "firebase serve --only functions",
+    "update-firebase-config": "firebase functions:config:set transak.public_key_staging=$(grep TRANSAK_PUBLIC_KEY_STAGING .env | cut -d '=' -f 2-) transak.private_key_staging=$(grep TRANSAK_PRIVATE_KEY_STAGING .env | cut -d '=' -f 2-) transak.public_key_prod=$(grep TRANSAK_PUBLIC_KEY_PROD .env | cut -d '=' -f 2-) transak.private_key_prod=$(grep TRANSAK_PRIVATE_KEY_PROD .env | cut -d '=' -f 2-) ramp.public_key_staging=$(grep RAMP_PUBLIC_KEY_STAGING .env | cut -d '=' -f 2-) ramp.public_key_prod=$(grep RAMP_PUBLIC_KEY_PROD .env | cut -d '=' -f 2-) moonpay.public_key_staging=$(grep MOONPAY_PUBLIC_KEY_STAGING .env | cut -d '=' -f 2-) moonpay.private_key_staging=$(grep MOONPAY_PRIVATE_KEY_STAGING .env | cut -d '=' -f 2-) moonpay.public_key_prod=$(grep MOONPAY_PUBLIC_KEY_PROD .env | cut -d '=' -f 2-) moonpay.private_key_prod=$(grep MOONPAY_PRIVATE_KEY_PROD .env | cut -d '=' -f 2-)",
```

Comment:
I've moved this to a separate file for better separation of concerns but this is one command rather than multiple commands strung together so it probably won't be much more readable.